### DISCUSSION TITLE
Swipable thumbnails

### DISCUSSION THREAD

**Discussion by Stanislavmin** [igordanchenko/yet-another-react-lightbox#193] [Q&A] [answered]
Hello, I am just curious if it is possible to implement in some way that thumbnails behave as in this [example](https://swiperjs.com/demos#thumbs-gallery-loop): 

A lot of customers try to "swipe" the thumbnails once they see them(mobile). Maybe there are css I can adjust? I understand that there is a fundamental difference in how thumbinals are built (nav + buttons in yarl vs div with images in swiper) but curious if there is something we can do to work around it :) 

Best regards

**Comment by igordanchenko** [comment, marked as answer]
Hi there! 

Thank you for sharing this feedback. I'm afraid there is no magical css property that would turn current implementation into completely different one :) 

1) It should be possible to add some sort of swipe-able behavior to the current Thumbnails plugin implementation. I've been considering it at some point, but eventually postponed it due to already non-trivial animation effects. This could be a good project if someone is willing to contribute.

2) It's also possible to implement the desired UX as a separate plugin. I'll definitely consider adding it to this repo if someone is willing to contribute a PR.

**Reply by elminmd** [reply]
hello @igordanchenko  i want to ask , has anyone did this PR ? :)

**Reply by igordanchenko** [reply]
I'm afraid there have been no volunteers.

**Reply by elminmd** [reply]
thanks for answer
